### ISSUE TITLE
CognitoServiceIdentityProvider errors

### ISSUE THREAD

**Issue by pilipicus** [iann0036/former2#130]
Everything was perfectly working last August that I used Former2 but today I'm getting a few different "CognitoServiceIdentityProvider " errors, without being able to get any element from the scanning.
I reinstalled the webApp and the Chrome browser extension, doing all the changes in the files suggested in the guide (https://github.com/iann0036/former2/blob/master/HOSTING.md
), but I'm getting always the same errors.

**Comment by iann0036** [comment]
Hi @pilipicus,

Could you let me know the exact text of these errors? You can find the detail (assuming Chrome) in View -> Developer -> JavaScript Console and look for the red text.

Also, could you confirm which region you're scanning?

**Comment by pilipicus** [comment]
The region set is EU(Ireland).

The errors that appear seems being linked to the Former2 Helper extension but I modified the files correctly like I did in the past (bg.js and manifest.json in the extension, app.js in the webApp).:

Skipping IoT1ClickDevicesService.listDevices NetworkingError
mappings.js:2 Error calling WorkSpaces.describeConnectionAliases. "The call to the SDK failed. You may need to update the Former2 Helper extension."
mappings.js:2  Too many requests for CognitoIdentityServiceProvider.adminListGroupsForUser, sleeping for 679ms
mappings.js:2 Too many requests for CognitoIdentityServiceProvider.adminGetUser, sleeping for 3780m
mappings.js:2 Too many requests for CognitoIdentityServiceProvider.adminListGroupsForUser, sleeping for 998ms

**Comment by iann0036** [comment]
Thanks for that @pilipicus,

I believe the only log here that would cause a visible error here is this one:

```
mappings.js:2 Error calling WorkSpaces.describeConnectionAliases. "The call to the SDK failed. You may need to update the Former2 Helper extension."
```

The rest are informational and can be safely ignored.

The error that alerted is probably a result of an older AWS SDK version used in the browser extension. New additions constantly come in to the application so the SDK does need updating, however I've neglected to update the SDK packages in the former2-helper repo. Sorry for that.

I've updated the former2-helper repo with the latest SDK now, so if you reapply the changes with the latest commit there (or apply the newer SDK to your existing), that might fix some of those errors you're seeing.

Let me know how that goes.
